man/docker.1.md

```diff
@@ -425,6 +425,26 @@ device.
 
 Example use: `docker daemon --storage-opt dm.use_deferred_removal=true`
 
+#### dm.use_deferred_deletion
```

Comment:
@rhvgoyal If it should be on in the general case why isn't it enabled by default?

The last point was my attempt to explain to the reader when to use this feature. You can also try:

"You should enable deferred device deletion on the device mapper if you regularly mount the directory of one container into another container.  This feature also helps when unintentional device leaks across mount namespaces are possible. For example, because a user did `unshare -m` or a special volume mount (`-v `/`:/host`). "